modules/nf-core/sentieon/datametrics/main.nf

```diff
@@ -0,0 +1,87 @@
+process SENTIEON_DATAMETRICS {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    container 'nf-core/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta)
```

Comment:
The `meta2` and `meta3` in the input does not seem to be used. Is it best-practice to include them anyway? Here is an example where it is included for some "fields" but not for others:

https://github.com/nf-core/modules/blob/6f150e1503c0826c21fedf1fa566cdbecbe98ec7/modules/nf-core/ensemblvep/vep/main.nf#L10-L17

Perhaps the best thing is to ask in the modules Slack-channel 🤔